compiler/il/OMRILOps.hpp

```diff
@@ -883,6 +883,22 @@ class ILOpCode
       return TR::BadILOp;
       }
 
+   static TR::ILOpCodes indirectStoreOpCode(TR::DataType type)
```

Comment:
There already is a function for this: https://github.com/eclipse/omr/blob/5330053254111a4e41c4afeabd53c7a5e5c16034/compiler/il/OMRIL.cpp#L646

Reply:
I'm not sure what the intent was for those, but those ones aren't static.

Reply:
OK.  The exercise to just use a single, static implementation is not as trivial as I had hoped as it will require coordinated changes with OpenJ9 and likely some other adjustments to the code.